frontend/src/scenes/App.tsx

```diff
@@ -111,7 +117,15 @@ function AppScene(): JSX.Element | null {
         (activeScene ? loadedScenes[activeScene]?.component : null) ||
         (() => (showingDelayedSpinner ? <SceneLoading /> : null))
 
-    const toastContainer = <ToastContainer autoClose={8000} transition={Slide} position="bottom-right" />
+    const toastContainer = (
+        <ToastContainer
+            autoClose={8000}
```

Comment:
These are critical attributes for this kind of component. The amount of time the toast is show, the animation speed and duration as it moves in and out of view, etc. Typically I cap these durations at 7 seconds or less. The slide transition might make these feel slower in some ways - were there any other transition or animation types you tried that felt just as effective but possibly snappier of faster?